cmake/faceliftMacros.cmake

```diff
@@ -136,67 +136,99 @@ function(facelift_generate_code )
     set(multiValueArgs IMPORT_FOLDERS)
     cmake_parse_arguments(ARGUMENT "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
+    # generate only interfaces which has changed since last build
```

Comment:
"Call the code generator if either the interface or the code generator has changed"